sites/intelligent-aerospace/server/templates/index.marko

```diff
@@ -34,25 +34,79 @@ $ const adSlots = {
 
   <div class="row">
     <div class="col-lg-8">
-      <endeavor-content-query-feed
-        header="Featured"
-        limit=7
-        skip=5
-        section-id=section.id
-        native-x={ placement: 'list2', aliases, index: 6 }
-      />
+      <div class="row">
+        <div class="col-lg-6 mb-block">
+          <endeavor-content-query-section-list
+            limit=3
+            section-alias="commercial"
+            header={ title: "Commercial", href: "/commercial" }
+            native-x={ placement: 'list1', aliases: ['commercial'], index: 2 }
+          />
+        </div>
+        <div class="col-lg-6 mb-block">
+          <endeavor-content-query-section-list
+            limit=3
+            section-alias="military"
+            header={ title: "Military", href: "/military" }
+            native-x={ placement: 'list1', aliases: ['military'], index: 2 }
+          />
+        </div>
+      </div>
+      <div class="row">
+        <div class="col-lg-6 mb-block">
+          <endeavor-content-query-section-list
+            limit=3
+            section-alias="avionics"
+            header={ title: "Avionics", href: "/avionics" }
+            native-x={ placement: 'list1', aliases: ['avionics'], index: 2 }
+          />
+        </div>
+        <div class="col-lg-6 mb-block">
+          <endeavor-content-query-section-list
+            limit=3
+            section-alias="atc-atm"
+            header={ title: "ATC/ATM", href: "/atc-atm" }
+            native-x={ placement: 'list1', aliases: ['atc-atm'], index: 2 }
+          />
+        </div>
+      </div>
     </div>
     <div class="col-lg-4 ad-rail">
       <endeavor-gam-ad-unit-display id="gpt-ad-rail1" />
       <endeavor-gam-ad-unit-display id="gpt-ad-rail2" />
     </div>
   </div>
-
+  <div class="row">
+    <div class="col-lg-4 mb-block">
+      <endeavor-content-query-section-list
+        limit=3
+        section-alias="helicopter"
+        header={ title: "Helicopter", href: "/helicopter" }
+        native-x={ placement: 'list1', aliases: ['helicopter'], index: 2 }
+      />
+    </div>
+    <div class="col-lg-4 mb-block">
+      <endeavor-content-query-section-list
+        limit=3
+        section-alias="satcom"
+        header={ title: "SATCOM", href: "/satcom" }
+        native-x={ placement: 'list1', aliases: ['satcom'], index: 2 }
+      />
+    </div>
+    <div class="col-lg-4 mb-block">
+      <endeavor-content-query-section-list
+        limit=3
+        section-alias="unmanned"
+        header={ title: "Unmanned", href: "/unmanned" }
+        native-x={ placement: 'list1', aliases: ['unmanned'], index: 2 }
+      />
+    </div>
+  </div>
   <div class="row">
     <div class="col-lg-4 mb-block">
       <theme-pennwell-published-content-query-list
         query={
-          contentTypes: ["Video"],
+          contentTypes: ["Video", "Webinar"],
```

Comment:
I'm assuming you don't want to include Webinars with Videos?

Reply:
Removed, good catch.